spec/services/allocation_service_spec.rb

```diff
@@ -124,50 +124,33 @@
     end
   end
 
-  describe '#allocations' do
-    it "Can get allocations by prison", vcr: { cassette_name: 'prison_api/allocation_service_get_allocations_by_prison' } do
-      first_offender_id = 'JSHD000NN'
-      second_offender_id = 'SDHH87GD'
-      leeds_prison = 'LEI'
+  describe '#previously_allocated_poms' do
```

Comment:
As commented in the implementation: I'm not sure this method is needed or used by anyone? We already have `Allocation#previously_allocated_poms` which is very similar.